packages/workspace/src/api/CreateWorkspaceRequest.ts

```diff
@@ -1,4 +1,11 @@
+import { API } from '@capsulajs/capsulajs-configuration-service';
+
 export default interface CreateWorkspaceRequest {
   /** Token used to get workspace configuration */
   token: string;
+  /**
+   * The type of configuration provider, that will be used to get configuration
```

Comment:
missing options what is the providers

Reply:
@idanilt We use the type ConfigurationProvider, where all of these options are described. Do you want to also dublicate it in the description of this file?

Reply:
at least put a link
it was very hard to find it

Reply:
@idanilt Sorry, I've noticed that docs don't automatically take the types from an external modules to show it. I will include the link, you're right.

Reply:
I removed the block but please resolve it

Reply:
Ok, I've added the text with possible values.